### PR TITLE
docs: :memo: Add details on settings to use for protected branches, plus other fixes

### PR DESCRIPTION
**Pull request by lwjohnst86**
I got rid of the doc about tags and releases, I'll add that in another doc later once we've gotten to that point.

### CODE REVIEW

File: entries/admin.md
Comment (signekb) [inline entries/admin.md:1]: Is this file missing an introduction? I'm not sure what this is / what this entry should cover :)

File: entries/admin.md [hunk 1]
```diff
@@ -3,25 +3,25 @@ title: "General admin content"
...
-sending an update document to our funder or stakeholders.
-
 ## GitHub repository settings
 
 -   Always
     [protect](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/managing-a-branch-protection-rule)
     the main branch of the repository to limit forced pushes and
     deletions.
+- Branch protection settings to enable are:
+    - Require approval of at least 2
```
Comment (signekb) [inline entries/admin.md:13]: at least 2 *developers* or *team members*

File: entries/admin.md [hunk 2]
```diff
@@ -3,25 +3,25 @@ title: "General admin content"
...
+    - Dismiss approvals when new commits are added
```
Comment (signekb) [inline entries/admin.md:14]: Not sure what this means? :)

File: entries/admin.md [hunk 3]
```diff
@@ -3,25 +3,25 @@ title: "General admin content"
...
+    - Allow Team Leader to bypass requirements
+    - Require approval of most recent reviewable push
+    - Require status checks to pass
+    - Require branches to be up to date
+    - Restrict who can push to the `main` branch.
```
Comment (signekb) [inline entries/admin.md:19]: Who is able to push to the main branch? Only team lead?

File: entries/admin.md [hunk 4]
```diff
@@ -3,25 +3,25 @@ title: "General admin content"
...
 
 ## GitHub settings
 
 Whenever a new GitHub repo is created for a new website or software, run this Terminal command on it in order to set some default options for the newly created repository.
 
 ``` bash
-gh repo edit --delete-branch-on-merge=true --enable-wiki=false --enable-discussions=false
+gh repo edit --delete-branch-on-merge=true --enable-wiki=false --enable-discussions=false --enable-auto-merge=true --allow-update-branch
```
Comment (signekb) [inline entries/admin.md:26]: I haven't done this for sprout. Have you, @lwjohnst86 ? :)

**Review by signekb — CHANGES_REQUESTED** [review]
I'm a bit confused about the purpose of this "admin" entry. I think it needs an introduction :)

**Review by K-Beicher — COMMENTED** [review]
I may still be a bit muddled, but I can't make heads or tails of this.  Where does it fit and what is it...?

File: entries/admin.md (hunk 1 above)
Comment (lwjohnst86) [reply entries/admin.md]: I think 2 might be too much... but we can assess as we go along.

File: entries/admin.md (hunk 2 above)
Comment (lwjohnst86) [reply entries/admin.md]: These are options within the protected branches section. I've added a bit of details to the reasons for adding these settings.

File: entries/admin.md (hunk 4 above)
Comment (lwjohnst86) [reply entries/admin.md:26]: I have :relaxed: for all the repos

File: entries/admin.md [hunk 5]
```diff
@@ -3,24 +3,24 @@ title: "General admin content"
...
+    - "Dismiss stale approvals when new commits are added"
+    - "Allow specified actors to bypass requirements", the actor being the team leader
+    - "Require approval of most recent reviewable push"
+    - "Require status checks to pass before merging"
+    - "Restrict who can push to matching branches" and "Restrict pushes that create matching branches" for the `main` branch
 
 ## GitHub settings
 
-Whenever a new GitHub repo is created for a new website or software, run this Terminal command on it in order to set some default options for the newly created repository.
+Whenever a new GitHub repo is created for a new website or software project, run this Terminal command on it in order to set some default options for the newly created repository. These options set up the repository with things that we need (or don't need, like the wiki). For instance, delete branches after they've been merged, allow PR's to have an option to auto-merge after approval, and for PR's to have an option to easily update with the `main` branch.
```
Comment (signekb) [inline entries/admin.md:23]:
```suggestion
Whenever a new GitHub repo is created for a new website or software project, run this Terminal command on it in order to set some default options for the newly created repository. These options set up the repository with things that we need (and omit things we don't need, like a wiki). 

The command sets up the following settings: 
- Delete branches after they've been merged
- Omit wiki
- Disable discussions
- Allow PR's to have an option to auto-merge after approval
- Allow PR's to have an option to easily update with the `main` branch.
```

**Review by signekb — COMMENTED** [review]
Nice! It's much clearer to me now. 
However, I'm not sure that the headers **GitHub repository settings** and **GitHub settings** are that meaningful? E.g., under GitHub settings, the text is about whenever a new *repo* is created?

**Review by K-Beicher — DISMISSED** [review]
Looks like a sensible adjustment to the points.

**Comment by lwjohnst86** [comment]
Is this better @signekb?

**Review by K-Beicher — APPROVED** [review]
Makes much more sense now.

**Pull request merged**